src/components/common/SmartPrice.tsx

```diff
@@ -33,7 +33,10 @@ export const SmartPrice: React.FC<Props> = ({ buyToken, sellToken, price: priceF
       price = buyOrderPriceInverse
       priceInverse = buyOrderPrice
     }
-    return [formatPrice(price), formatPrice(priceInverse)]
+
+    // set decimals to 5 places - default is 4 from @dex-js
+    // TODO: consider changing dex-js default to 5
+    return [formatPrice({ price, decimals: 5 }), formatPrice({ price: priceInverse, decimals: 5 })]
```

Comment:
Ok, si this in this version too? ok if it's safe
anyways, pls, lets not use magic numbers. Lets add some constant with a semantic name

Reply:
:+1: